### ISSUE TITLE
Sort apps on last access

### ISSUE THREAD

**Issue by kasnder** [OxfordHCC/tracker-control-android#121]
This would greatly improve the UX

**Comment by kasnder** [comment]
This could be integrated into #151

**Comment by wYhyzByH** [comment]
I'd like to see this feature as well, but not generally by last traffic seen, but for apps with blocked traffic.
So having the option to sort the main view not just by name or number of blocked companies, but also by recently blocked ones would ease the troubleshooting of things going on right now.
Currently if I install a new app and things are getting blocked and the app is not working as expected, I always have to find it in the long list by searching its name, before I can adjust the app specific settings.
To see it in the top 10 apps directly on the start screen would be great.

**Comment by kasnder** [comment]
Yeah, this makes complete sense! Thank you for the suggestion. :)

**Comment by user11111112** [comment]
I would like this too, please!  While the default sorting-by-category approach and then chronologically within those categories is great for apps with few trackers, my web browser has hundreds of trackers collected from every website I visit, and when a new site won't load it takes some time and searching to find the offending tracker.  If all trackers could be sorted chronologically (with category indicators) then it would be very straightforward to start troubleshooting the issue and get back on task.

Thank you for all of your hard work on this amazing app!!!
